core/src/main/scala/akka/kafka/internal/TransactionalProducerStage.scala

```diff
@@ -258,6 +300,12 @@ private final class TransactionalProducerStageLogic[K, V, P](
     }
   }
 
+  private val onCommitFailedAckCb: AsyncCallback[Throwable] = {
+    getAsyncCallback[Throwable](
+      t => super.onCompletionFailure(t)
```

Comment:
The `super` is hideous, should it be just `failStage(t)`?

Reply:
We could, but if we ever implement anything in `DefaultProducerStage.onCompleteFailure` we may want it to execute that.